RoomDraw/app/views/users/show.html.erb

```diff
@@ -1,4 +1,5 @@
 <div class="center">
-  <h1 class="center">User Login Success!</h1>
-  <p class="center">User <%= current_user.student_id %> has been successfully logged in</p>
+  <h1 class="center">Welcome <%= (Student.find current_user.student_id).name %></h1>
+  <% room = student_room(current_user.student_id) %>
+  <h3><%= if room.present? then "Your room is #{room.dorm_name}: ##{room.room_num}" else "You have not yet drawn a room" end %></h3>
```

Comment:
While this certainly works, it feels like it might be worth including more information---e.g. Consider that Kevin and I are in a double. It it might be nice for it to say `You are in #{room.dorm_name}: ##{room.room_num}` alongside similar information for the other groupmates you might have. It would be a nice way of letting people know that what they drew in to, they drew in to as part of group X which contains members Y and that they'll be in rooms M, N, and O.